### PR TITLE
update-profile

### PR DESCRIPTION
**Pull request by Abhi-gits**

### CODE REVIEW

File: components/team/MainTeam.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ const MainContent: React.FC = () => {
             Our team <b className='text-green-700 '>Flora Finder</b>
           </h1>
           <p className="text-red-500 text-lg font-light">
-            With over 100 years of combined experience, we&apos;ve got a well-seasoned team at the helm.
+            With over 50+ years of combined experience, we&apos;ve got a well-seasoned team at the helm.
```
Comment (tech-bash) [inline components/team/MainTeam.tsx:15]: Do we have 50 years of experience sir?

File: components/team/Team.tsx [hunk 2]
```diff
@@ -80,7 +65,7 @@ const Team: React.FC = () => {
             title: "Programmer & Web Developer",
             imageSrc: "/assets/profile/subrat.png",
             linkedinUrl: "https://www.linkedin.com/in/subratyadav/",
-            twitterUrl: "https://twitter.com/findingsubrat?t=hUHGhRBlKT2ZKJfcmPXaGQ&s=09",
+            //twitterUrl: "https://twitter.com/findingsubrat?t=hUHGhRBlKT2ZKJfcmPXaGQ&s=09",
```
Comment (anurag629) [inline components/team/Team.tsx]: remove this line

File: components/team/TeamMember.tsx [hunk 3]
```diff
@@ -62,9 +62,9 @@ const TeamMember: React.FC<TeamMemberProps> = ({
           <a href={linkedinUrl} target="_blank" rel="noopener noreferrer" className="flex rounded-full h-7 w-7 hover:scale-150 transition-transform duration-300">
             <FontAwesomeIcon icon={faLinkedin} className="text-blue-500 mx-auto mt-2" />
           </a>
-          <a href={twitterUrl} target="_blank" rel="noopener noreferrer" className="flex rounded-full h-7 w-7 hover:scale-150 transition-transform duration-300">
+          {/* <a href={twitterUrl} target="_blank" rel="noopener noreferrer" className="flex rounded-full h-7 w-7 hover:scale-150 transition-transform duration-300">
```
Comment (anurag629) [inline components/team/TeamMember.tsx]: remove this line

**Review by anurag629 — APPROVED** [review]
remove the twitter comments and merge it

File: components/team/MainTeam.tsx (hunk 1 above)
Comment (Abhi-gits) [reply components/team/MainTeam.tsx:15]: yes( 5 * 6) + 20+(mentors experience)

**Pull request merged**